coordinator/tests/services/translation/response/release/release-response.spec.ts

```diff
@@ -9,19 +8,23 @@ import * as fs from "fs"
 import * as path from "path"
 import {getUniqueValues} from "../../../../../src/utils/collections"
 import {resolveOrganization, resolvePractitioner, toArray} from "../../../../../src/services/translation/common"
-import {hl7V3} from "@models"
+import {fhir, hl7V3} from "@models"
 import {
   getHealthcareServices,
   getLocations,
+  getMedicationRequests,
+  getMessageHeader,
   getOrganizations,
+  getPatient,
   getPractitionerRoles,
-  getPractitioners
+  getPractitioners,
+  getProvenances
 } from "../../../../../src/services/translation/common/getResourcesOfType"
 import {getRequester, getResponsiblePractitioner} from "../common.spec"
 
 describe("outer bundle", () => {
   const result = createOuterBundle(getExamplePrescriptionReleaseResponse())
-  console.log(LosslessJson.stringify(result))
+  LosslessJson.stringify(result)
```

Comment:
Don't think this call is doing anything

Reply:
If it's there to check that the result can be stringified, I'd use:
```
expect(() => LosslessJson.stringify(result)).not.toThrow()
```

Reply:
yea not really sure why its there in the first place
i just removed all the console logs to stop tests being so annoying
think ill remove all the stringifies except the top level to ensure everything can be stringified.